src/app/models/session.rb

```diff
@@ -140,9 +140,8 @@ def estimated_interest
       # We therefore add some ghost "ballast votes" across the board to all sessions, so as to
       # make estimated_interest tend toward the mean in cases when there are few real votes.
 
-      ballast_votes = 3.0
-
-      (session_votes + ballast_votes) / (possible_votes + ballast_votes * session_count) * participant_count
+      ballast_votes = 10.0
```

Comment:
Got there by manual fiddling. 3 was giving wild results for just-created sessions with very few votes. But estimate vs. actual attendance analysis suggests this whole method makes very little difference & needs rethinking.